pkgs/development/python-modules/hypothesmith/default.nix

```diff
@@ -1,65 +1,52 @@
 {
   lib,
+  black,
   buildPythonPackage,
   fetchPypi,
   hypothesis,
   lark,
   libcst,
   parso,
   pytestCheckHook,
+  pytest-cov-stub,
   pytest-xdist,
   pythonOlder,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "hypothesmith";
   version = "0.3.3";
-  format = "setuptools";
+  pyproject = true;
 
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
 
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-lsFIAtbI6F2JdSZBdoeNtUso0u2SH9v+3C5rjOPIFxY=";
   };
 
-  patches = [ ./remove-black.patch ];
+  build-system = [ setuptools ];
 
-  postPatch = ''
-    substituteInPlace setup.py \
-      --replace "lark-parser" "lark"
-
-    substituteInPlace tox.ini \
-      --replace "--cov=hypothesmith" "" \
-      --replace "--cov-branch" "" \
-      --replace "--cov-report=term-missing:skip-covered" "" \
-      --replace "--cov-fail-under=100" ""
-  '';
-
-  propagatedBuildInputs = [
+  dependencies = [
     hypothesis
     lark
     libcst
   ];
 
   nativeCheckInputs = [
+    black
```

Comment:
We don't want to rely on linting tools like black.

Reply:
It's not used for linting hypothesmith's code. It's used to lint the code generated by it afaict.